### DISCUSSION TITLE
Change the default app icon

### DISCUSSION THREAD

**Discussion by ratasorin** [remix-run/remix#4958] [Q&A] [answered]
I would like to change the default `favicon.ico` icon that comes with a Remix app. How should I do that ?

**Comment by sergiodxa** [comment, marked as answer]
Replace the icon file in the public folder, your browser may take time to update due caching but that should be enough

**Reply by artemis-prime** [reply]
Why no mention of the icon links in metadata?  When I include those, the browser still seems to try to load `public/favicon.icon`... why?  Should the metadata values be respected?
